pkg/apis/core/v1alpha1/types_propagationpolicy.go

```diff
@@ -120,6 +123,16 @@ const (
 	SchedulingModeDivide SchedulingMode = "Divide"
 )
 
+// ReplicasStrategy determines the strategy used by the scheduler when assigning replicas.
+// +kubebuilder:validation:Enum=PreferenceBinpack
+type ReplicasStrategy string
+
+const (
+	// ReplicasStrategyPreferenceBinpack means the replicas will be divided into clusters
+	// as few as possible, while considering clusters' capacity during the division.
+	ReplicasStrategyPreferenceBinpack ReplicasStrategy = "PreferenceBinpack"
```

Comment:
I think preferred is better than preference.

Reply:
+1

Reply:
Actually, `binpack` would do.

Reply:
+1. I like the brief one.